contracts/script/Deploy.s.sol

```diff
@@ -1,57 +1,39 @@
-// SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.22;
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.26;
 
-import "forge-std/Script.sol";
-import {SP1Blobstream} from "../src/SP1Blobstream.sol";
+import {Script, console} from "forge-std/Script.sol";
+
+import {FuelStreamX} from "../src/FuelStreamX.sol";
 import {ERC1967Proxy} from "@openzeppelin/proxy/ERC1967/ERC1967Proxy.sol";
 import {SP1MockVerifier} from "@sp1-contracts/SP1MockVerifier.sol";
 import {ISP1Verifier} from "@sp1-contracts/ISP1Verifier.sol";
 
-import {BaseScript} from "./Base.s.sol";
-
 // Required environment variables:
-// - SP1_PROVER
 // - GENESIS_HEIGHT
 // - GENESIS_HEADER
-// - SP1_BLOBSTREAM_PROGRAM_VKEY
-// - CREATE2_SALT
+// - VKEY
 // - SP1_VERIFIER_ADDRESS
 
-contract DeployScript is BaseScript {
+contract DeployScript is Script {
     function setUp() public {}
 
-    string internal constant KEY = "SP1_BLOBSTREAM";
-
-    function run() external multichain(KEY) returns (address) {
+    function run() public {
         vm.startBroadcast();
 
-        SP1Blobstream lightClient;
-        ISP1Verifier verifier = ISP1Verifier(
-            vm.envOr("SP1_VERIFIER_ADDRESS", 0x3B6041173B80E77f038f3F2C0f9744f04837185e)
-        );
-
-        // Deploy the SP1Blobstream contract.
-        SP1Blobstream lightClientImpl =
-            new SP1Blobstream{salt: bytes32(vm.envBytes("CREATE2_SALT"))}();
-        lightClient = SP1Blobstream(
-            address(
-                new ERC1967Proxy{salt: bytes32(vm.envBytes("CREATE2_SALT"))}(
-                    address(lightClientImpl), ""
-                )
-            )
-        );
-
-        // Initialize the Blobstream X light client.
-        lightClient.initialize(
-            SP1Blobstream.InitParameters({
-                guardian: msg.sender,
-                height: uint32(vm.envUint("GENESIS_HEIGHT")),
-                header: vm.envBytes32("GENESIS_HEADER"),
-                blobstreamProgramVkey: vm.envBytes32("SP1_BLOBSTREAM_PROGRAM_VKEY"),
-                verifier: address(verifier)
-            })
-        );
-
-        return address(lightClient);
+        // The verifier, handled by Succinct Labs
+        ISP1Verifier verifier = ISP1Verifier(vm.envAddress("SP1_VERIFIER_ADDRESS"));
+
+        // Deploy the FuelStreamX contract.
+        uint64 trustedHeight = uint64(vm.envUint("GENESIS_HEIGHT"));
+        bytes32 trustedBlockHeader = bytes32(vm.envBytes("GENESIS_HEADER"));
+        bytes32 vkey = bytes32(vm.envBytes("VKEY"));
+
+        FuelStreamX fuelStreamX =
+            new FuelStreamX(trustedHeight, trustedBlockHeader, vkey, address(verifier));
```

Comment:
Are you deploying the FuelStreamX contract temporarily until we connect with the real FuelStreamX contract on mainnet? If that is the case I suggest that you rename this to something like `contracts/script/DeployDev.s.sol` or something like that. The idea is to have different scripts eventually, depending on what and where to deploy.

Reply:
or is this just a script to deploy ethereum contracts whenever we need? In that case ignore my suggestion.

Reply:
The contract will be handled by the fuel team, this is just an example